include/Evaluation_und_Validation.tex

```diff
@@ -52,14 +52,38 @@ \subsection{Begründung des Nichtumsetzens von 8.6}
+
+\subsection{Sprints und Sprintreviews/Planning}
+Die Unterteilung in Zweiwöchige Sprints war rückblickend gesehen die richtige Entscheidung. Eine längere Sprintzeit hätte uns Flexibilität gekostet, während bei einwöchigen Sprints davon auszugehen ist, dass der Überblick auf das Gesamtprojekt verloren gehen würde und generell mehr Stress und mehr indirekte Kosten (im Bezug auf verbrauchte Zeit durch Planung und Review) entstanden wären. Die Reviewsitzungen dauerten durchschnittlich ein bis zwei Stunden und dienten zur Überprüfung der erledigten Arbeiten, Planung des nächsten Vorgehens und zur Teambildung. Die investierte Zeit half zwar nicht produktiv für das Endprodukt, waren aber für den Erfolg des Projektes vital und wird daher als erfolgreich gewertet.
+
+\subsection{Kommunikation}
+Durch das ganze Projekt hindurch wurde ein proaktiver Ansatz für die Kommunikation gewählt, sowohl zum Kunden und unserem Betreuer, wie auch teamintern. Verwendet wurde dafür Mail, WhatsApp und Telefon. Nach Rückmeldungen wurde diese offene und zeitnahe Kommunikation vom Kunden sehr geschätzt. Innerhalb des Teams wurde nach einem harzigen Sprint, indem die Kommunikation zum erlegen kam, einen wöchentlichen Austausch am Sonntagabend eingeplant. Dieser half erstens, dass man auch über weiche Probleme reden konnte, wie auch, dass man über den Fortschritt und die Planung des anderen informiert wurde. Wir schätzen diesen Punkt als extrem wichtig und werten ihn als erfolgreich.
+
+\subsection{Versuche}
+Den methodischen Ansatz an die Versuche mit einer vorgängigen Versuchsplanung, Berichterstattung während der Durchführung und anschliessender Auswertung entspricht dem gängigen Vorgehen, war für uns jedoch Neuland. Rückblickend gab es uns Schritt für Schritt vor, was zu tun ist und schaffte dadurch Sicherheit und eine höhere Vertrauenswürdigkeit in die gewonnenen Resultate. Wir würden dieses Vorgehen empfehlen und werten es als erfolgreich.
+
+\subsection{Protokolle}
+Jedes Meeting mit dem Kunden wurde aufgezeichnet und anschliessend in ein annähernd wörtliches Protokoll niedergeschrieben. Dieses Protokoll half uns auch nach einem Monat wichtige Details noch präsent zu haben und war daher ein wichtiges Hilfsmittel. Einziger Nachteil dabei war der hohe Arbeitsaufwand, der mit der Erstellung des Protokolls verbunden war, eine automatisierte Lösung wäre daher vorzuziehen. Wir schätzen es dennoch als erfolgreich für das Projekt ein.
```

Comment:
Ein grosser Nachteil war der hohe Arbeitsaufwand, welcher mit der erstellung dieses Protokolles einhergeht. Durch den enormen Zeitaufwand betrachten wir diese Protokolle als Teilerfolg. Da der Zeitaufwand in der Industrie nicht verhältnismässig wäre, würden wir empfehlen in einem Zukünftigen Projekt eher ein Entscheidungsprotokoll zu erstellen. Als erfolgreich betrachten wir jedoch die Aufnahmen, da dadurch auch Analysiert werden kann, wie Missverständnisse entstanden sind.

(irgendwie finde ich eben die Preisleistung des Protokolles als nicht gut, da zu viel zeit für nichts verbratet wurde)

Reply:
Aber du findsch nüt ide Uufnahme, Text isch durchsuechbar